docs/Advanced/Refs.mdx

```diff
@@ -86,11 +86,11 @@ export interface CanvasRef {
   /**
    * Zoom in on the canvas.
    */
-  zoomIn?: () => void;
+  zoomIn?: (zoomFactor: number) => void;
```

Comment:
requiring a zoomFactor makes this a breaking change. could we default it to something so we don't have to break existing apis ?

Reply:
oops yea that should be optional